### ISSUE TITLE
How do I set up the configuration to switch from training a Lora for Lumina Image 2.0 to actually finetuning the model ？please!

### ISSUE THREAD

**Issue by samliu315** [tdrussell/diffusion-pipe#276]
@tdrussell 
As the title says.

**Comment by alienzed** [comment]
What is your goal? Lowering the LR?

Look into force_constant_lr in your config toml

```
...
# Learning rate warmup.
warmup_steps = 100
force_constant_lr = 3e-5
...
```

**Comment by samliu315** [comment]
> What is your goal? Lowering the LR?

I think you replied to the wrong issue

**Comment by alienzed** [comment]
Well training and fine tuning are generally differentiated by the learning rate or What you are training. You can finetune a LORA just as you can finetune a Model as well, apologies if my reply was not helpful.

**Comment by samliu315** [comment]
I figured it out and tried to follow the instructions in the model support doc: "delete or comment out the [adapter]", and it worked. It was very easy.
